.github/workflows/ci.yaml

```diff
@@ -54,10 +60,66 @@ jobs:
       TRIAL_BRANCH: ${{ needs.metadata.outputs.TRIAL_BRANCH }}
     secrets: inherit
 
+  publish-mealkit:
+    needs: [metadata, amd64, arm64]
+    if: "!canceled()" && github.event_name == 'schedule' && needs.metadata.outputs.PUBLISH == 'true'
+    strategy:
+      fail-fast: true
+      matrix:
+        - TAG: JAX
+          TARGET_IMAGE: jax
+        - TAG: PALLAS
+          TARGET_IMAGE: pallas
+        - TAG: PAX
+          TARGET_IMAGE: upstream_pax
+        - TAG: T5X
+          TARGET_IMAGE: upstream_t5x
+        - TAG: ROSSETA_PAX
+          TARGET_IMAGE: pax
+        - TAG: ROSSETA_T5X
+          TARGET_IMAGE: T5x
+    uses: ./.github/workflows/_publish_container.yaml
+    with:
+      SOURCE_IMAGE: |
+        ${{ needs.amd64.outputs.TAG_[format('{0}_MEALKIT', matrix.TAG)] }}
+        ${{ needs.arm64.outputs.[format('{0}_MEALKIT', matrix.TAG)] }}
+      TARGET_IMAGE: ${{ matrix.TARGET_IMAGE }}
+      TARGET_TAGS: |
+        type=raw,value=mealkit,priority=500
+        type=raw,value=mealkit-${{ needs.metadata.outputs.BUILD_DATE }},priority=500
+
+  publish-final:
+    needs: [metadata, amd64, arm64]
+    if: "!cancelled()" && github.event_name == 'schedule' && needs.metadata.outputs.PUBLISH == 'true'
```

Comment:
Same comment about `github.event_name == 'schedule'` being included in `needs.metadata.outputs.PUBLISH`